packages/runtime/container-runtime/src/summary/summarizerHeuristics.ts

```diff
@@ -157,16 +157,21 @@ export class SummarizeHeuristicRunner implements ISummarizeHeuristicRunner {
 	}
 
 	public shouldRunLastSummary(): boolean {
-		const opsSinceLastAck = this.opsSinceLastAck;
+		const weightedOpsSinceLastAck = getWeightedNumberOfOps(
+			this.heuristicData.numRuntimeOps,
+			this.heuristicData.numNonRuntimeOps,
```

Comment:
Are nonRuntimeOps user ops or is the other way around?

Reply:
Runtime ops are the new "user" ops. This was changed in https://github.com/microsoft/FluidFramework/pull/10463

Reply:
Unfortunately, that PR is not very descriptive and its hard to tell what happened, but I get the gist :-).